src/Entity/EthereumServer.php

```diff
@@ -129,8 +135,53 @@ public function validateConnection() {
         ]),
       ];
     }
-
     return $return;
   }
 
+  /**
+   * Server info as render Array Table.
+   *
+   * @return array
+   *    Table render array.
+   */
+  public function getServerInfoAsTable() {
```

Comment:
As mentioned above, we shouldn't put these kinds of methods on the config entity, but in their specific controller instead.